config/locales/hyrax.en.yml

```diff
@@ -71,6 +71,7 @@ en:
           header:         "Edit Administrative Set"
         form:
           cancel:               "Cancel"
+          note:                 "Users granted a new role will only see works that are deposited after that role has been granted."
```

Comment:
What if they are granted "approver" but they are already a "viewer"?  Then "see works" won't be correct.  How about "...will only gain the role on works that are deposited..."